code/modules/hydroponics/grown/misc.dm

```diff
@@ -203,6 +203,8 @@
 	max_integrity = 40
 	wine_power = 80
 	discovery_points = 300
+	var/fusetime = 40 //in deciseconds
+	var/fusevariance = 10
```

Comment:
Underlined variables and use seconds defines
fuse_time
fuse_variance

Reply:
Variables define-ified and deciseconds decimated